CPAC/pipeline/cpac_pipeline.py

```diff
@@ -347,8 +344,8 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
     Setting MKL_NUM_THREADS to 1
     Setting ANTS/ITK thread usage to {ants_threads}
     Maximum potential number of cores that might be used during this run: {max_cores}
-
-"""
+{random_state}
```

Comment:
Does the `random_state` or `random_seed` go here?

E.g. in numpy:
```python
> import numpy as np
> np.random.seed(1234)
> np.random.get_state()
('MT19937',
 array([      1234, 3159640283, 4062961311, 3954462607, 3112783424,
        2849714703, .... ]], dtype=uint32),
 624,
 0,
 0.0)

```

Reply:
Here `random_state` is a named placeholder that fills in with https://github.com/FCP-INDI/C-PAC/blob/4073ab32416f2a13ee157f197a7ebf94f4522870/CPAC/pipeline/cpac_pipeline.py#L374-L377
so it says `    Random seed: {seed}` on that line in the log if a seed is set and just puts an empty string there otherwise. (I believe the scope of the variable is just within that string and its `format` method)

For example, with `--random_seed 1` I get

```
220121-16:10:13,317 nipype.workflow INFO:
	 
    Run command: run /media/ebs/C-PAC-1.8.1-testing /media/ebs/random_seed_check/seed1 participant --save_working_dir --data_config_file /configs/data_config_regtest.yml --pipeline_file /config/random_seed.yml --mem_gb 100 --n_cpus 8 --num_ants_threads 8 --participant_ndx 3 --random_seed 1

    C-PAC version: 1.8.3.gb0a7cce3-dev

    Setting maximum number of cores per participant to 8
    Setting number of participants at once to 1
    Setting OMP_NUM_THREADS to 1
    Setting MKL_NUM_THREADS to 1
    Setting ANTS/ITK thread usage to 8
    Maximum potential number of cores that might be used during this run: 8
    Random seed: 1
```

and with no random seed set, I get

```
    Run command: run /media/ebs/C-PAC-1.8.1-testing /media/ebs/random_seed_check/unspecified1 participant --save_working_dir --data_config_file /configs/data_config_regtest.yml --pipeline_file /config/random_seed.yml --mem_gb 100 --n_cpus 8 --num_ants_threads 8 --participant_ndx 3

    C-PAC version: 1.8.3.gb0a7cce3-dev

    Setting maximum number of cores per participant to 8
    Setting number of participants at once to 1
    Setting OMP_NUM_THREADS to 1
    Setting MKL_NUM_THREADS to 1
    Setting ANTS/ITK thread usage to 8
    Maximum potential number of cores that might be used during this run: 8
```

Reply:
Makes sense, thanks!